frontend/src/components/offers/offers.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Tab,Tabs } from '@mui/material'; 
+import { useEffect, useState } from "react";
+import productService from '../../services/product.service';
+import Card from '../card/card';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import "./offers.css"
+function Offers() {
+  var tab = 1; 
+  const [products,setProducts] = useState()
+  const count = 8;
+  const [start,setStart] = useState(0)
+  const[more,setMore] = useState(true);
```

Comment:
Watch for spaces.
Can we be more consistent when using semicolons ? 
Altough it is not necessary for code, it is much prettier to have on each line and we can avoid some potential errors lates.